Week1/homework/js-exercises/Exercise 10/compareArrays.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+const myArray = ['naji', 'nabulsi', 46, { 'Married': true }];
+const myIteams = ['laptop', 'shoes', 'book', 'Ben', 'Story', 'bag', 'glasses'];
+console.log('The length of the array is...' + ' ' + myArray.length);
```

Comment:
Why do you add the space like this, you can just append it to the first string like this
`'The length of the array is... ' + myArray.length`